en/docs/learn/username-recovery-via-user-preferred-notification-channel.md

```diff
@@ -0,0 +1,353 @@
+# Username Recovery Via User Preferred Notification Channel
+
+WSO2 Identity Server enables recovering user accounts where the user has forgotten the user name.
+
+!!! Note
+    This feature is only available via Account Recovery REST APis. Currently, WSO2 IS does not 
+    support this feature via the User Portal.    
+
+**Username Recovery Flow**
+
+ - User provides a set of unique claims to identify the user account.
+ - Then the server will prompt available notification channels to the user to receive username 
+ recovery notification.
+ - User selects a notification channel.
+ - Sever send the recovery notification to the user via the preferred notification channel.
+
```

Comment:
- The user provides..

- Next, the server prompts the user to select from the available notification channels for receiving the username recovery notification. 

- The user...

- The server sends the ..